### PR TITLE
Use target architecture for building ANCM symbol packages.

### PR DESCRIPTION
**Pull request by jkotalik**
Infrastructure change to make sure ANCM symbols are built for x86.

### CODE REVIEW

**Review by Tratcher — APPROVED** [review]

**Comment by anurse** [comment]
Since the PDBs end up "shipping" I'll run this through ask mode, but I anticipate this should be straightforward.

**Comment by anurse** [comment]
So, wait for Tactics approval for merging.

**Comment by anurse** [comment]
Given this is indeed an infrastructure-only change. Go ahead and merge under tell-mode.

**Pull request merged**
